core/src/main/java/bio/terra/pearl/core/service/participant/ParticipantUserService.java

```diff
@@ -14,7 +15,7 @@
 import java.util.UUID;
 
 @Service
-public class ParticipantUserService extends ImmutableEntityService<ParticipantUser, ParticipantUserDao> {
```

Comment:
Is this a drive-by change?
Not sure I understand how this fits in here.

Reply:
Good question -- I needed to be able to update the lastLogin value of an existing user using this service, which means it needs the `update` method from CrudService.  Previously, updates to lastLogin were only done through CurrentUserService, and so the fact that this inherited from the wrong class (ParticipantUser objects are definitely not immutable) wasn't noticed